src/pages/CompanyCourseManagement/CCMPage.jsx

```diff
@@ -0,0 +1,97 @@
+// Import necessary components from Material-UI library
+import Button from "@mui/material/Button";
+import { List, ListItem, ListItemText } from "@mui/material";
+// Import custom PageLayout component
+import PageLayout from "../../components/PageLayout";
+
+// CCMPage component definition
+function CCMPage() {
+  // Return JSX markup for the CCMPage component
+  return (
+    // Use the custom PageLayout component with specific title and viewport settings
+    <PageLayout title="CCM" viewportPage>
+
+      {/* Container for the page content */}
+      <div
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          justifyContent: "center",
+          height: "100vh",
+          padding: "20px", 
+        }}
+      >
+
+        {/* Button for adding training */}
```

Comment:
Please already provide an empty onClick Function

Reply:
// Empty onClick function for the button
  const handleAddTrainingClick = () => {
    console.log("Add Training button is clicked");
  };
  
  ^ Taken from latest commit.
  Hope this is enough for this request